Server/DeviceManagement/DeviceManagement.Application/CQRS/Commands/CreateUserCommand.cs

```diff
@@ -0,0 +1,6 @@
+﻿using System;
+using ModularHouse.Libraries.InternalMessaging.CQRS.Abstractions.Command;
+
+namespace ModularHouse.Server.DeviceManagement.Application.CQRS.Commands;
+
+public record CreateUserCommand(Guid Id) : ICommand;
```

Comment:
Make data more specific. Rename `Id` to `UserId`;
So we have not `command.Id`, but `command.UserId`